src/elm/Network.elm

```diff
@@ -250,7 +250,11 @@ viewSummaryWidgets ctx remoteNodes =
                   , icon = Icons.lastBlock iconSize
                   , value =
                         RemoteData.map
-                            (\nodes ->
+                            (\nodesDict ->
+                                let
+                                    compatibleNodes =
+                                        Dict.filter (\_ node -> node.client >= ctx.statsVersion) nodesDict
```

Comment:
If at some point we have node version 10 then those nodes would be filtered out here.

Reply:
Indeed.

I'll add a note to the readme, but I don't want to deal with this now, seeing that it will happen in years.

Reply:
I factored all these checks into a single function, and added a comment to that function.